packages/api/src/orbitAPI.ts

```diff
@@ -12,58 +12,32 @@ import { BlobLike } from "./genericHTTPAPI";
 import { RequiredSpec } from "./util/requiredSpec";
 
 // Meant to conform to genericHTTPAPI/Spec, but I can't declare conformance without running into obscure Typescript limitations.
-type SpecLegacy = {
-  "/taskStates": {
-    GET: {
+export type ValidatableSpec = {
+  "/taskStates"?: {
+    GET?: {
       query:
         | ({
+            /**
+             * @minimum 1
+             * @TJS-type integer
```

Comment:
Cannot specify a default here due to a [limitation with `ajv`](https://ajv.js.org/guide/modifying-data.html#assigning-defaults). Defaults cannot be assigned within a `anyOf`.